components/mpas-ocean/driver/ocn_comp_mct.F

```diff
@@ -2761,6 +2766,15 @@ subroutine ocn_export_mct(o2x_o, errorCode)   !{{{
         call mpas_pool_get_array(statePool, 'accumulatedFrazilIceMass', accumulatedFrazilIceMass, 1)
      end if
 
+     ! Cryo fields
+     if (config_land_ice_flux_mode == 'standalone' .or. 'data') then
```

Comment:
And here.

Reply:
ok I get it, I made a lot of mistakes!

Reply:
Sorry for laying it on.  I just wanted to make sure it all got caught.